README.md

```diff
@@ -47,7 +47,7 @@ The Subnet EVM runs in a separate process from the main AvalancheGo process and
 [v0.5.0] AvalancheGo@v1.10.0 (Protocol Version: 25)
 [v0.5.1] AvalancheGo@v1.10.1-v1.10.4 (Protocol Version: 26)
 [v0.5.2] AvalancheGo@v1.10.1-v1.10.4 (Protocol Version: 26)
-[v0.5.3] AvalancheGo@v1.10.5-v1.10.6 (Protocol Version: 27)
+[v0.5.3] AvalancheGo@v1.10.5-v1.10.8 (Protocol Version: 27)
```

Comment:
it's not directly related. indirectly I realized these both issues during a report. Since it was just a readme change, I did not create a separate PR.